### PR TITLE
Fix non compliance route

### PR DESCRIPTION
**Pull request by etInfp**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [] In Progress
- [X ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by etInfp** [comment]
@bziser [bziser](https://github.com/bziser) Hello :) 
Can't figure out what the problem is here..

File: Packs/MISP/Integrations/MISPV3/MISPV3.yml [hunk 1]
```diff
@@ -2855,7 +2855,7 @@ script:
     description: Publish an event.
     execution: false
     name: misp-publish-event
-  dockerimage: demisto/pymisp2:1.0.0.27224
+  dockerimage: demisto/pymisp2:1.0.0.27777
```
Comment (abaumgarten) [inline Packs/MISP/Integrations/MISPV3/MISPV3.yml:2858]: Please merge from master, see here: https://github.com/demisto/content/pull/1832

**Comment by bziser** [comment]
@etInfp Please run `demisto-sdk update-release-notes -i Packs/Infinipoint -u (major|minor|revision|documentation)` to generate release notes

**Comment by etInfp** [comment]
@bziser [bziser](https://github.com/bziser)
Any updates ?

**Pull request merged**